public/app/core/components/QueryOperationRow/QueryOperationRow.tsx

```diff
@@ -92,6 +118,10 @@ const getQueryOperationRowStyles = stylesFactory((theme: GrafanaTheme) => {
       align-items: center;
       justify-content: space-between;
     `,
+    dragIcon: css`
+      opacity: 0.4;
+      cursor: hand;
+    `,
```

Comment:
Shouldn't it be `drag` instead of hand?  `cursor: hand;` is probably invalid (couldn't find it in [mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor))

Reply:
you are right! great catch